### ISSUE TITLE
client: create "kd-io" team for each environment during startup

### ISSUE THREAD

**Issue by rjeczalik** [koding/koding#9827]
Currently we create koding team per each env by default, we'd need also a `kd-io` team for kd.io, so it's created for each environment (dev, sandbox etc.). Production already has that team created (/cc @usirin)

**Comment by rjeczalik** [comment]
The team should have public registration.

`kd auth register` and `kd auth register --team foobar` should both register to that team by default (/cc @mehmetalisavas).

**Comment by gokmen** [comment]
yes, @usirin was proposing to use it as default, it would be better to define this team in config and provide it in default schema as well. we need to make it public but for the first release access to this group over koding will be limited or denied at all, it will be the abstract team for `kd.io` requirements.

**Comment by rjeczalik** [comment]
We should also add newly registered user to that team by default. And for old users either 1) add a test on login - if user is not a member of `kd-io`, add them or 2) perform a migration.
